### PR TITLE
HTTPSTB: apply Mike Jones' feedback

### PR DESCRIPTION
**Pull request by equalsJeffH**
HTTPSTB: apply Mike Jones' feedback: https://www.ietf.org/mail-archive/web/unbearable/current/msg01297.html

technical comments addressed:

> Section 5.2:  Change “the Token Binding ID in the identity token” to
> “the Token Binding ID (or a cryptographic hash of it) in the identity
> token”.
> 
> 
> 
> Section 5.2:  Change “include in the token the Token Binding ID” to
> “include in the token the Token Binding ID (or a cryptographic hash
> of it)”.

Others are editorial cleanups.

### CODE REVIEW

File: draft-ietf-tokbind-https-09.xml [hunk 1]
```diff
@@ -391,16 +389,19 @@
...
-        <t>The scoping for those Token Binding key pairs generated by Web browsers in 
-        the context of the first-party and federation use cases defined in this 
-        specification (below), and to be used for binding HTTP cookies MUST be at the 
-        granularity of "effective top-level domain (public suffix) + 1" (eTLD+1), 
-          i.e., at the same granularity at which cookies can be set 
-          (see <xref target="RFC6265"/>).  Key pairs used to bind other application
+        <t>The scoping of Token Binding key pairs generated by Web browsers for use in 
+        first-party and federation use cases defined in this 
+        specification (<xref target="federation"/>, below), and intended for 
+        binding HTTP cookies, MUST be at the
```
Comment (Andrei-Popov) [inline draft-ietf-tokbind-https-09.xml]: At IETF 98, we agreed to say that browsers absolutely cannot exceed the scope of eTLD+1, but can use a narrower scope if cookies are scoped more narrowly.

File: draft-ietf-tokbind-https-09.xml [hunk 2]
```diff
@@ -476,16 +477,16 @@
...
 	<t>To better protect the security of the identity token, the
 	Identity Provider may wish to bind the identity token to the TLS
 	connection between the client and the Relying Party, thus
-	ensuring that only said client can use the identity token: The
-	Relying Party will compare the Token Binding ID in the identity
-	token with the Token Binding ID of the TLS connection between it
-	and the client.</t>
+	ensuring that only said client can use the identity token; the
+	Relying Party will compare the Token Binding ID (or a cryptographic 
+	hash of it) in the identity token with the Token Binding ID of the
```
Comment (Andrei-Popov) [inline draft-ietf-tokbind-https-09.xml]: ... or hash:)

File: draft-ietf-tokbind-https-09.xml [hunk 3]
```diff
@@ -955,22 +957,22 @@ contexts. Other approaches are possible, but are outside the scope of this speci
...
+          message sent by the client to the Token Consumer and any message
           sent by the Token Consumer to the client. The Token Consumer is the
           party that redirects the client to the Token Provider. In this case,
-          the man-in-the-middle controls the redirect URL, and can tamper with
+          the man-in-the-middle controls the redirect URL and can tamper with
           any redirect URL issued by the Token Consumer (as well as with any
           Javascript running in the origin of the Token Consumer). The goal of
-          the man-in-the-middle is to trick the Token Provider to issue a token
-          bound to <spanx style="emph">its</spanx> Token Binding ID, not to
+          the man-in-the-middle is to trick the Token Provider to issuing a token
```
Comment (Andrei-Popov) [inline draft-ietf-tokbind-https-09.xml]: Into issuing?

**Comment by Andrei-Popov** [comment]
Otherwise, looks good. Thanks, Jeff!

File: draft-ietf-tokbind-https-09.xml (hunk 1 above)
Comment (equalsJeffH) [reply draft-ietf-tokbind-https-09.xml]: done.

File: draft-ietf-tokbind-https-09.xml (hunk 2 above)
Comment (equalsJeffH) [reply draft-ietf-tokbind-https-09.xml]: done, i think.

File: draft-ietf-tokbind-https-09.xml (hunk 3 above)
Comment (equalsJeffH) [reply draft-ietf-tokbind-https-09.xml]: done, thx.

**Comment by equalsJeffH** [comment]
@Andrei-Popov's comments incorp'd -- thx!

**Pull request merged**